mbf_abstract_core/include/mbf_abstract_core/abstract_planner.h

```diff
@@ -84,8 +84,8 @@ namespace mbf_abstract_core
        *         71..99 are reserved as plugin specific errors
        */
       virtual uint32_t makePlan(const geometry_msgs::PoseStamped &start, const geometry_msgs::PoseStamped &goal,
-                                double tolerance, std::vector<geometry_msgs::PoseStamped> &plan, double &cost,
-                                std::string &message) = 0;
+                                double dist_tolerance, double angle_tolerance,
+                                std::vector<geometry_msgs::PoseStamped> &plan, double &cost, std::string &message) = 0;
```

Comment:
This is an interface change ;) This will break the API :-1:

Reply:
Yes, see my comment below

Reply:
Japp, I saw it after writing this ;)